lightkurve/correctors/backgroundcorrector.py

```diff
@@ -0,0 +1,139 @@
+"""Defines a BackgroundCorrector class which provides a simple way to correct a
+light curve by utilizing the pixel time series data contained within the
+target's own Target Pixel File. Specifically, this corrector is intended to be
+used on TESS target pixel files provided by TESSCut.
+
+PixelCorrector builds upon RegressionCorrector by correlating the light curve
+against a design matrix composed of the following elements:
+* A background light curve to capture the dominant scattered light systematics.
+* Background-corrected pixel time series to capture any residual systematics.
+* Splines to capture the target's intrinsic variability.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from . import DesignMatrix, DesignMatrixCollection
+from .regressioncorrector import RegressionCorrector
+from .designmatrix import create_spline_matrix
+from .. import MPLSTYLE
+
+
+__all__ = ['BackgroundCorrector']
+
+
+class BackgroundCorrector(RegressionCorrector):
```

Comment:
If you are removing more than the background (i.e. correcting the pixel level systematics à la PLD) then this is not really a `BackgroundCorrector`, it is more of a `TESSFFICorrector`. Can we consider a better name?

Reply:
I think I'd be happy to go with `TessFFICorrector`.  We want there to be a good, robust, default corrector for TESS FFI's, so we can use this name for whatever corrector we find is the best default!

Reply:
I'm inclined to go ahead and call this `TessPLDCorrector`, because the type of method being used is best known as PLD.